Scripts/Static maps.R

```diff
@@ -73,16 +74,19 @@ rurality <- population_density %>%
 
 # Define colour palettes
 #pal_1 <- colorBin(bin_colours, population_density$pop_density, bins = 7, pretty = T)
-pal_1 <- colorNumeric("magma", population_density$pop_density)
-pal_2 <- colorFactor("magma", population_density$ur8_2020_name)
+pal_1 <- colorNumeric("magma", population_density$pop_density_log)
+pal_2 <- rev(colorFactor("magma", population_density$ur8_2020_name)) # Reverse colour scheme to match population density
```

Comment:
this errors for me, can use reverse = T in colorFactor instead
```suggestion
pal_2 <- colorFactor("magma", population_density$ur8_2020_name, reverse = T) # Reverse colour scheme to match population density
```